docs/user-guide/node-selection/index.md

```diff
@@ -198,3 +198,194 @@ must be satisfied for the pod to schedule onto a node.
 
 For more information on inter-pod affinity/anti-affinity, see the design doc
 [here](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/podaffinity.md).
+
+## Taints and tolerations (beta feature)
+
+### Specifying taints and tolerations
+
+Node affinity, described earlier, allows a pod to "be attracted to" a set of nodes
+(either as a preference or a hard requirement). Taints are the opposite -- they allow
+a *node* to *repel* a set of pods.
+
+More conceretely: *taints* go on nodes, and *tolerations* go on pods. A taint on a node
+means "repel all pods that do not toleate the taint."
+
+You add a taint to a node using `kubectl taint`. For example,
```

Comment:
Could you please add a link to `kubectl taint` command here https://kubernetes.io/docs/user-guide/kubectl/kubectl_taint/

I have updated it a bit by adding `NoExecute`: 
1) https://github.com/kubernetes/kubernetes.github.io/pull/2744
2) https://github.com/kubernetes/kubernetes/pull/42775